conda_forge_tick/update_upstream_versions.py

```diff
@@ -447,30 +522,52 @@ def _update_upstream_versions_process_pool(
 ) -> None:
     futures = {}
     with executor(kind="dask", max_workers=10) as pool:
-        for node, node_attrs in gx.nodes.items():
+        _all_nodes = [t for t in gx.nodes.items()]
+        random.shuffle(_all_nodes)
+        for node, node_attrs in _all_nodes:
             with node_attrs["payload"] as attrs:
                 if attrs.get("bad") or attrs.get("archived"):
                     attrs["new_version"] = False
                     continue
                 futures.update(
-                    {pool.submit(get_latest_version, attrs, sources): (node, attrs)},
+                    {pool.submit(get_latest_version, node, attrs, sources): (
+                        node, attrs)},
                 )
+
+        n_tot = len(futures)
```

Comment:
Might be of some value to consider using something like tqdm here?

Reply:
I’ve found that tqdm output in CI services is not super great.